doc/topics/releases/nitrogen.rst

```diff
@@ -220,6 +223,37 @@ release cycle (two major releases after this one), those who are using the
 :ref:`netapi module <all-netapi-modules>`) are encouraged to update their code
 to use ``tgt_type``.
 
+``full_return`` Argument in ``LocalClient`` and ``RunnerClient``
+----------------------------------------------------------------
+
+An ``full_return`` argument has been added to the ``cmd`` and ``cmd_sync``
+methods in ``LocalClient`` and ``RunnerClient`` which causes the return data
+structure to include job meta data such as ``ret_code`` and ``success``.
```

Comment:
Should this be `retcode` instead of `ret_code`? 

I also don't see anything in the example return data in the examples below about the `success` key in the `cmd.run` return dicts.